eskip/string.go

```diff
@@ -103,15 +105,23 @@ func (r *Route) backendString() string {
 
 // Serializes a route expression. Omits the route id if any.
 func (r *Route) String() string {
+	return r.Print(false)
+}
+
+func (r *Route) Print(pretty bool) string {
```

Comment:
sorry for not being completely clear. I meant a). See `func (r *Route) String()` string and `func String(routes ...*Route) string`. There could be a `func Print(routes ...*Route) string`, too.